chapter2/tests/t20/terraform/base/main.tf

```diff
@@ -0,0 +1,14 @@
+module "label" {
```

Comment:
move the data source to vpc.tf
module label - to proper tf file where it's used for the first time and then delete this empty main.tf file

Reply:
> move the data source to vpc.tf module label - to proper tf file where it's used for the first time and then delete this empty main.tf file

@dmitry-mightydevops
Done